src/Directory.Build.props

```diff
@@ -25,6 +25,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="PolySharp" Version="1.10.0" Condition="'$(TargetFramework)' != 'net5' AND '$(TargetFramework)' != 'net6' AND '$(TargetFramework)' != 'net7'">
```

Comment:
Isn't the point here that PolySharp determines what polyfills are necessary based on which .NET version is being referenced?

```suggestion
    <PackageReference Include="PolySharp" Version="1.10.0">
```

Reply:
It does indeed. This condition should not be necessary.

Reply:
You may try. I tried it at first ang got compilation errors.

Reply:
OK, I got the reason.